<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options1</class>
 <widget class="QWidget" name="Options1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 64, 65)</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>60</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopInsulinButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>1161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>45</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(192, 28, 40)</string>
   </property>
   <property name="text">
    <string>STOP INSULIN</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sleepButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>1161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>45</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string>Sleep Screen</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="activityButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>1161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>45</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string>Activity</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="myPumpButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>1161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>45</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string>My Pump</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
